lib/console_display.rb

```diff
@@ -9,8 +10,11 @@ def initialize(game, console)
 
   def start
     show_start_screen
-    while !@game.board.full?
-      play_round
+    while !@game.over?
+      play_turn
+      show_board
+      game_drawn?
```

Comment:
be careful with naming here. `game_drawn?` and `game_won?` have side effects where they print a message to the screen when the game is over. This also goes back to having [command/query separation](http://martinfowler.com/bliki/CommandQuerySeparation.html). I would maybe make this a little bit more explicit.

You could remove `game_drawn?` and `game_won?`, and outside the while loop just have `display_outcome`.